server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -383,8 +377,27 @@ private CommandExecutor startCommandExecutor(
         return executor;
     }
 
+    @Nullable
+    private SessionManager initializeSessionManager() {
+        final AuthConfig authCfg = cfg.authConfig();
+        if (authCfg == null) {
+            return null;
+        }
+        SessionManager manager;
+        try {
+            manager = new FileBasedSessionManager(new File(cfg.dataDir(), "_sessions").toPath(),
+                                                  authCfg.sessionValidationSchedule());
+            manager = new CachedSessionManager(manager, Caffeine.from(authCfg.sessionCacheSpec()).build());
+            return new ExpiredSessionDeletingSessionManager(manager);
+        } catch (IOException e) {
+            throw new IOError(e);
```

Comment:
Can we just propagate the exceptions as they are and the `CompletableFuture` of `start()` is completed with the exception?

Reply:
Fixed